libs/sdk-react-native/android/src/main/java/com/breezsdk/BreezSDKModule.kt

```diff
@@ -238,11 +238,11 @@ class BreezSDKModule(reactContext: ReactApplicationContext) : ReactContextBaseJa
     }
 
     @ReactMethod
-    fun receivePayment(amountSats: Double, description: String, promise: Promise) {
+    fun receivePayment(reqData: ReceivePaymentRequest, promise: Promise) {
```

Comment:
React Native only allows certain argument types to be passed over the RN bridge from javascript <-> native: https://reactnative.dev/docs/native-modules-android#argument-types, so this needs to be `reqData: ReadableMap`

Reply:
Done in 3b9ef403befd5079c2e761d6dacd0f0beda0e125